Source/UnrealEnginePython/UnrealEnginePython.Build.cs

```diff
@@ -13,9 +13,9 @@ protected string PythonHome
     {
         get
         {
-            return Path.GetFullPath(Path.Combine(ModuleDirectory, "..", "..", pythonHome));
-        }
-    }
+			return Path.GetFullPath(Path.Combine(ModuleDirectory, "../../../",  pythonHome));
```

Comment:
I'm not sure the intention for this change but it has broken compiling for me in Win64. The only way this will work is if the python folder is put directly in the Plugins folder and not under the UnrealEnginePython folder.